test/src/test/java/org/locationtech/geowave/test/query/SpatialTemporalQueryIT.java

```diff
@@ -446,6 +462,32 @@ private static void ingestTimeRangeData(
 		writer.write(feature);
 	}
 
+	private static void ingestTimeRangeDataForDuplicateDeletion(
+			final Calendar cal,
+			final Writer writer,
+			final SimpleFeatureBuilder featureTimeRangeBuilder,
+			final int min,
+			final int max,
+			final int field,
+			final String name )
+			throws IOException {
+		final GeometryFactory geomFactory = new GeometryFactory();
+		final int midPoint = (int) Math.floor((min + max) / 2.0);
+		cal.set(
+				field,
+				min);
+		featureTimeRangeBuilder.add(geomFactory.createPoint(new Coordinate(
+				0,
+				0)));
+		featureTimeRangeBuilder.add(cal.getTime());
+		cal.set(
+				field,
+				max);
+		featureTimeRangeBuilder.add(cal.getTime());
+		SimpleFeature feature = featureTimeRangeBuilder.buildFeature(name + ":fullrange");
```

Comment:
Changed the name to avoid headaches down the road. This was one of those concepts that was a bit fuzzy to me originally (which data needs to be unique), technically it still worked. But, queries with dataId would fail in this case.